tests/python/unittest/test_ndarray.py

```diff
@@ -1099,6 +1099,39 @@ def test_assign_float_value_to_ndarray():
     b[0] = a[0]
     assert same(a, b.asnumpy())
 
+@with_seed()
+def test_ndarray_assignment():
```

Comment:
1. Add issue link as code comment.
2. Change the function name to a special one since this is only for testing special cases. General cases have been covered by `test_ndarray_indexing`.

Reply:
Okay. Thanks!